library/src/scripts/messages/messageStyles.ts

```diff
@@ -95,6 +95,10 @@ export const messagesClasses = useThemeCache(() => {
         zIndex: 20,
     });
 
+    const fullWidth = style("fullWidth", {
```

Comment:
Instead of using these styles, I'd rather we optionally used the <Container> component, which already limits the width of the contents.

Reply:
Fixed- Wrap message component inside container component.